src/nccl_ofi_rdma.c

```diff
@@ -7281,6 +7287,29 @@ int nccl_net_ofi_rdma_init(const char *provider_filter,
 		goto error;
 	}
 
+	if (!nccl_ofi_dmabuf_viable())
```

Comment:
Not sure why you need the goto here instead of just putting the newly added block in a `if (nccl_ofi_dmabuf_viable())` block